tools/modules/system/manage_dtoverlays.sh

```diff
@@ -35,6 +35,23 @@ function manage_dtoverlays () {
 		else
 			available_overlays=$(ls -1 ${overlaydir}/${overlay_prefix}*.dtbo | sed 's/^.*\('${overlay_prefix}'.*\)/\1/g' | sed 's/'${overlay_prefix}'-//g' | sed 's/.dtbo//g')
 		fi
+
+		# Check the branch in case it is not available in /etc/armbian-release
+		if [[ -z "${BRANCH}" ]]; then
```

Comment:
Not sure about here. I've added these codes because i realized that BRANCH is not exists on /etc/armbian-release

Reply:
Yes, we determine BRANCH at first run script. When kernel is changing via armbian-config, this might not be the case. Needs to be investigated.

Reply:
@igorpecovnik can you check again